packages/@react-stately/table/docs/useTableColumnResizeState.mdx

```diff
@@ -0,0 +1,41 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-stately/table';
+import {ClassAPI, HeaderInfo, FunctionAPI, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-stately/table/package.json';
+
+---
+category: Collections
+keywords: [table, state, grid]
+---
+
+# useTableColumnResizeState
```

Comment:
Feels more discoverable to have this as a separate page. `useTableColumnResize` feels ok to have in `useTable` docs since it modifies the original implementation and thus needs context